### DISCUSSION TITLE
New and updated wiki

### DISCUSSION THREAD

**Discussion by RandomNinjaAtk** [RandomNinjaAtk/docker-lidarr-extended#36] [Announcements]
**Main Page:**
https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki

**Audio Information:**
https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Audio-Script-Information

**Video Information:**
https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Video-Script-Information

**Comment by Protektor-Desura** [comment]
I recommend you add to the wiki that if users want to easily import missing stuff from Discogs to MusicBrainz then they can use GreaseMonkey to do it much easier.
https://www.greasespot.net/
then use this plugin
https://github.com/murdos/musicbrainz-userscripts#discogs_importer

This will allow them to easily add missing music without too much trouble that exists in Discogs.
https://www.discogs.com/

**Reply by RandomNinjaAtk** [reply]
Good idea, I changed the wiki permissions, can you try adding it...

**Reply by Protektor-Desura** [reply]
Done let me know what you think.
https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Audio-Script-Information#q-i-see-my-missing-music-in-discogs-but-seems-complicated-to-add-it-to-musicbrainz-any-suggestions

**Reply by Protektor-Desura** [reply]
Opps. Typo fixed.

**Reply by RandomNinjaAtk** [reply]
Looks good!

**Reply by Protektor-Desura** [reply]
Thanks

**Reply by Yosh82** [reply]
Thank you! Glad I saw this post as have started adding missing albums to Musicbrainz with Picard this morning which is quite tedious, but they are on Discogs already so I can use the plugin!
